app/main.py

```diff
@@ -1,13 +1,32 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
+        "line": error["line_number"],
+        "column": error["column_number"],
+        "message": error["text"],
+        "name": error["code"],
+        "source": "flake8"
+    }
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+    return {
+        "errors": [
+            {
+                "line": error["line_number"],
+                "column": error["column_number"],
+                "message": error["text"],
+                "name": error["code"],
+                "source": "flake8"
+            }
```

Comment:
The code block from lines 14 to 20 duplicates the logic of the 'format_linter_error' function. According to the checklist, you should use the 'format_linter_error' function to avoid code duplication.

Reply:
fixed a problem